cmd/plume/release.go

```diff
@@ -275,16 +282,36 @@ func doGCE(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 			gceWaitForImage(pending)
 		}
 	} else {
-		obj := src.Object(src.Prefix() + spec.GCE.Image)
-		if obj == nil {
-			plog.Fatalf("GCE image not found %s%s", src.URL(), spec.GCE.Image)
-		}
-
 		if releaseDryRun {
 			plog.Noticef("Would create GCE image %s", name)
 			return
 		}
 
+		// temporary upload the image on GCS. It's required to create the Image to have a GCS URL.
+		imgURL := fmt.Sprintf("%s%s", spec.SourceURL(), spec.GCE.Image)
+
+		// verify key is set to "" to use the embedded one.
+		if err := sdk.DownloadSignedFile(spec.GCE.Image, imgURL, &http.Client{}, ""); err != nil {
+			plog.Fatalf("downloading GCE image from webserver: %v", err)
+		}
+
+		f, err := os.Open(spec.GCE.Image)
+		if err != nil {
+			plog.Fatalf("opening GCE image: %v", err)
+		}
+
+		defer f.Close()
+
+		o := gs.Object{Name: src.Prefix() + spec.GCE.Image}
+		if err := src.Upload(ctx, &o, f); err != nil {
```

Comment:
This this overwrite an existing object?

Reply:
Good question - it seems that default behavior is to leave early if there is an existing object matching a checksum.
https://github.com/flatcar/mantle/blob/a40798269e6c621d2f6fe4a1af27c6bd921ad72c/storage/bucket.go#L276-L277

We could do something like that:
```suggestion
                src.WriteAlways(true)
		if err := src.Upload(ctx, &o, f); err != nil {
```

Reply:
Seems yes

Reply:
Good, makes sense to set WriteAlways